src/github/pullRequestGitHelper.ts

```diff
@@ -27,121 +27,81 @@ export interface PullRequestMetadata {
 export class PullRequestGitHelper {
 	static ID = 'PullRequestGitHelper';
 	static async createAndCheckout(repository: Repository, pullRequest: IPullRequestModel) {
```

Comment:
how about renaming this something like `checkoutFromFork`?

Reply:
+1. done